cos_registration_server/applications/migrations/0004_alter_lokialertrule_rules_and_more.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.10 on 2025-01-23 11:58
```

Comment:
we don't want two migrations since the model addon. combine everything in one. Maybe commit the migration once this pr is about to be done.